src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -3,7 +3,7 @@ import {View} from 'react-native';
 import styles from '../../../styles/styles';
 import SidebarLinks from './SidebarLinks';
 import CreateMenu from '../../../components/CreateMenu';
-import FAB from '../../../components/FAB';
+import FAB from '../../../components/Fab';
```

Comment:
I think our preference is to keep FAB as an all caps name to indicate that it's an acronym (Floating Action Button)

Reply:
Updated accordingly.